dashel/dashel-common.cpp

```diff
@@ -59,83 +59,67 @@
 namespace Dashel
 {
 	using namespace std;
-	
+
 	// frome dashe-private.h
-	ExpandableBuffer::ExpandableBuffer(size_t size) :
-		_data((unsigned char*)malloc(size)),
-		_size(size),
-		_pos(0)
-	{
-	}
-	
+	ExpandableBuffer::ExpandableBuffer(size_t size) : _data((unsigned char*)malloc(size)), _size(size), _pos(0) {}
+
 	ExpandableBuffer::~ExpandableBuffer()
 	{
 		free(_data);
 	}
-	
+
 	void ExpandableBuffer::clear()
 	{
 		_pos = 0;
 	}
-	
+
 	void ExpandableBuffer::add(const void* data, const size_t size)
 	{
 		if (_pos + size > _size)
 		{
 			_size = max(_size * 2, _size + size);
 			_data = (unsigned char*)realloc(_data, _size);
 		}
-		memcpy(_data + _pos, (unsigned char *)data, size);
+		memcpy(_data + _pos, (unsigned char*)data, size);
```

Comment:
I find this version too cramped.

Reply:
what is doable is : 
 `memcpy(_data + _pos, (unsigned char *) data, size);` or  `memcpy(_data + _pos, ( unsigned char * ) data, size);`

Reply:
I like the first version.